firstvoices/backend/views/sites_views.py

```diff
@@ -32,18 +37,28 @@
             404: OpenApiResponse(description="Todo: Not Found"),
```

Comment:
Unrelated, but could you please switch the rest of these error messages to doc_strings while you're in here?